src/lib/Event/Listener/LoginListener.php

```diff
@@ -74,6 +74,15 @@ public function onSecurityInteractiveLogin(InteractiveLoginEvent $event): void
             return;
         }
 
+        $customerId = $this->configResolver->getParameter(
+            'authentication.customer_id',
+            Parameters::NAMESPACE
+        );
```

Comment:
This is code duplication from here: https://github.com/ezsystems/ezrecommendation-client/blob/2.1/src/lib/Event/Listener/LoginListener.php#L107-L110

It will be better to extract this to separate method e.g.: getCustomerId and use it both places.